### PR TITLE
Initial commit documentation for HTTPChannel AutoDecompress

### PR DESCRIPTION
**Pull request by mrsaldana**

### CODE REVIEW

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
Resolves #352

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -147,4 +147,5 @@ http.proxyredirect.httpPort.desc=The (non-secure) port to redirect from. Incomin
 http.proxyredirect.httpsPort=HTTPS port
 http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTTP requests that use the HTTP port are redirected to this port.
 
-
+http.autoDecompression=Auto decompression
+http.autoDecompression.desc=Controls whether the HTTP Channel will automatically decompress incoming request body data.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Specifies whether the HTTP channel automatically decompresses incoming request body data.

**Comment by mrsaldana** [comment]
Running a new build to include the second commit.
#build

**Comment by mrsaldana** [comment]
Build had one test fail, which is unrelated to the change made and did not show up in previous build. Seems like an environment issue. Merging branch.

**Pull request merged**